example/bioimageio-core-usage.ipynb

```diff
@@ -60,41 +69,91 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
```

Comment:
I don't know if you want to keep outputs, but I'd at least run the notebook once from the beginning. Seems like cells executed 3-4 are lost somewhere, 5 seems to be out of order, this bears the danger that the output doesn't reflect linear flow through the notebook. Also there seem to be errors in the outputs.

Reply:
I still need to fix the example model before the notebook can run again, but this is pending issues on the website... See also this comment: https://github.com/bioimage-io/core-bioimage-io-python/pull/229#issuecomment-1049908507.

Reply:
ah I see